data/building-hours/1-2-pause-kitchen.yaml

```diff
@@ -6,9 +6,10 @@ schedule:
   - title: Daylight
     notes: The late night menu starts at 9 p.m.
     hours:
-      - {days: [Mo, Tu, We, Th], from: '10:30am', to: '12:00am'}
-      - {days: [Fr, Sa], from: '10:30am', to: '9:00pm'}
-      - {days: [Su], from: '10:30am', to: '12:00am'}
+      - {days: [Mo, We], from: '10:30am', to: '12:00am'}
+      - {days: [Fr], from: '10:30am', to: '9:00pm'}
+      - {days: [Sa], from: '11:30am', to: '9:00pm'}
+      - {days: [Su, Tu, Th], from: '11:30am', to: '12:00am'}
```

Comment:
@ElijahVerdoorn, @mattk410

Do you two have any opinions? I can't decide which I like better.